point_cloud_client/src/lib.rs

```diff
@@ -19,12 +21,27 @@ impl PointCloudClient {
                     .map(|octree| *octree)
             })
             .collect();
+        let mut aabb = Aabb3::zero();
+        if let Ok(octrees) = &octrees {
```

Comment:
How about collapsing these ifs with `if let Ok((first, rest)) = octrees().ok().and_then(|octs| oct.split_first())`?

Reply:
Actually, can we use `?` in the expression above and here and bail early?

Reply:
I cleaned up the code, PTAL.